presubmit.py

```diff
@@ -0,0 +1,28 @@
+import os
+from subprocess import check_output, PIPE
+
+print("\nRunning tests....\n")
+
+try:
+    check_output("npm run api-test", shell=True, stderr=PIPE)
```

Comment:
nice! however imagine if we had 10 steps for presubmit, this file would grow a significantly in size. what about if we had 100 steps? this file would be 400+ lines long 😮 

consider making functions so we can avoid being redundant:

```py
def run_test(test_name, command):
  """
  In here you should put the try/except as well as the check_output function.
  params:
  test_name - the name of the test e.g. lint, build, etc.
  command - the command to be run by check_output e.g. npm run frontend-test
  """
  pass

```